### PR TITLE
Document what exactly is shared when using --error-reporting

### PR DESCRIPTION
**Pull request by eliecharra**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #87 
| ❓ Documentation  | yes

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -6,6 +6,12 @@
 
 By default, driftctl checks for a new version remotely. To disable this behavior, use either the flag `--no-version-check` or the environment variable `DCTL_NO_VERSION_CHECK=true`.
 
+#### Error reporting
+
+When a crash occur in driftctl, we do not send any crash reports.
```
Comment (wbeuil) [inline doc/README.md:11]: `When a crash occurs ...`

File: doc/README.md [hunk 2]
```diff
@@ -6,6 +6,12 @@
...
+For debugging purposes, you could add `--enable-reporting` when running driftctl and crash data will be sent to us via [Sentry](https://sentry.io)
+Detail of reported data could be found [here](./cmd/flags/error-reporting.md)
```
Comment (wbeuil) [inline doc/README.md:13]: `Details of reported data can ...`

File: doc/README.md [hunk 3]
```diff
@@ -6,6 +6,12 @@
 
 By default, driftctl checks for a new version remotely. To disable this behavior, use either the flag `--no-version-check` or the environment variable `DCTL_NO_VERSION_CHECK=true`.
 
+#### Error reporting
+
+When a crash occur in driftctl, we do not send any crash reports.
+For debugging purposes, you could add `--enable-reporting` when running driftctl and crash data will be sent to us via [Sentry](https://sentry.io)
```
Comment (wbeuil) [inline doc/README.md:12]: `..., you can add ...`

File: doc/cmd/flags/error-reporting.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Error reporting
+
+Here is a list of data we retrieve when error reporting is enabled.
```
Comment (wbeuil) [inline doc/cmd/flags/error-reporting.md:3]: `Below is a list of ...`

File: doc/cmd/flags/error-reporting.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+* **date**: Event date
+* **os name**: Operating system (string, e.g. : "linux | mac | windows")
```
Comment (wbeuil) [inline doc/cmd/flags/error-reporting.md:6]: `Operating System`

File: doc/cmd/flags/error-reporting.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+* **release**: driftctl version (string, e.g. : "v0.2.2")
+* **server_name**: Your computer hostname (string, e.g. : "yourhostname")
+* **runtime version**: Golang version (string, e.g. : "go1.15.2")
+* **runtime infos**: Variables go_maxprocs, go_numcgocalls, go_numroutines
+* **packages**: Golang used packages and their versions
+* **stacktrace**: The error stack
+
+## Example of full output
+
+Below is and full example of a nil pointer crash reported
```
Comment (wbeuil) [inline doc/cmd/flags/error-reporting.md:18]: `Below is a full example of a nil pointer crash report`

File: doc/cmd/flags/error-reporting.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+* **architecture**: Your CPU kind (string, e.g. : "amd64 | i389")
+* **num_cpu**: Your cpu cound (int, e.g. : 8)
+* **release**: driftctl version (string, e.g. : "v0.2.2")
+* **server_name**: Your computer hostname (string, e.g. : "yourhostname")
+* **runtime version**: Golang version (string, e.g. : "go1.15.2")
+* **runtime infos**: Variables go_maxprocs, go_numcgocalls, go_numroutines
+* **packages**: Golang used packages and their versions
+* **stacktrace**: The error stack
+
+## Example of full output
```
Comment (wbeuil) [inline doc/cmd/flags/error-reporting.md:16]: Either `Example of a full report output`, or `Real case example` or simply `Example`

File: doc/cmd/flags/error-reporting.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
+# Error reporting
+
+Here is a list of data we retrieve when error reporting is enabled.
+
+* **date**: Event date
+* **os name**: Operating system (string, e.g. : "linux | mac | windows")
+* **architecture**: Your CPU kind (string, e.g. : "amd64 | i389")
```
Comment (wbeuil) [inline doc/cmd/flags/error-reporting.md:7]: `Architecture of your CPU`

File: doc/cmd/flags/error-reporting.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+* **num_cpu**: Your cpu cound (int, e.g. : 8)
```
Comment (wbeuil) [inline doc/cmd/flags/error-reporting.md:8]: `Number of cores of your CPU`

**Review by wbeuil — CHANGES_REQUESTED** [review]

File: doc/README.md (hunk 3 above)
Comment (wbeuil) [reply doc/README.md:12]: `--error-reporting`

**Comment by eliecharra** [comment]
Fixed your remarks @wbeuil

**Review by wbeuil — APPROVED** [review]
LGTM

**Pull request merged**